cypress/support/gutenberg/plugins/blockSnapshots.js

```diff
@@ -202,6 +251,30 @@ export const registerBlockSnapshots = alias => {
 		return modifiedFn( ...[ ...args, {} ] )
 	} )
 
+	/**
+	 * Overwrite `assertBlockContent` by passing `blockSnapshots` option
+	 */
+	 Cypress.Commands.overwrite( 'assertBlockContent', ( originalFn, ...args ) => {
+		function modifiedFn( ...passedArgs ) {
+			const options = passedArgs.pop()
+			// Since Cypress commands are asynchronous, we need to pass a separate object to originalFn to avoid directly mutating the options argument.
+			const optionsToPass = cloneDeep( options )
+			// No need to assert backend because typeBlock asserts it for us
+			optionsToPass.assertBackend = false
```

Comment:
Would it be okay to not check the backend anymore here since typeBlock already checks the backend for us? That would assume that the assertBlockContent command is only called after the typeBlock command.